src/plugin.json

```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
   "type": "datasource",
-  "name": "Checkmk for Cloud Edition",
```

Comment:
please revert those changes, thats also why the ci failed

Reply:
I was expecting its fail before the rebranding commit merge. Now its ok.